trillian/ctfe/handlers_test.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/golang/mock/gomock"
-	"github.com/google/certificate-transparency-go"
+	ct "github.com/google/certificate-transparency-go"
```

Comment:
The import alias 'ct' isn't being used in other diffs for this commit ...  is the import necessary at all?

Reply:
Yes; the code uses `ct`, which is the top-level package name (i.e. it doesn't match the last path component).  This makes it explicit, which helps tools like `gofmt`.

Reply:
OK, couldn't see it easily in the GH code review page.